src/nautilus/msr.c

```diff
@@ -28,18 +28,26 @@
 inline void 
 msr_write (uint32_t msr, uint64_t data)
 {
+    // RISCV HACK
```

Comment:
same as before, apply this hack at the user site. Might expose some further chances for abstractions that way